frontend/src/lib/components/ui/UsdValueBanner.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  import IC_LOGO_ROUNDED from "$lib/assets/icp-rounded.svg";
+  import TooltipIcon from "$lib/components/ui/TooltipIcon.svelte";
+  import { LEDGER_CANISTER_ID } from "$lib/constants/canister-ids.constants";
+  import { icpSwapUsdPricesStore } from "$lib/derived/icp-swap.derived";
+  import { i18n } from "$lib/stores/i18n";
+  import { formatNumber } from "$lib/utils/format.utils";
+  import { nonNullish } from "@dfinity/utils";
+
+  export let usdAmount: number | undefined;
+
+  const absentValue = "-/-";
+
+  let usdAmountFormatted: string;
+  $: usdAmountFormatted = nonNullish(usdAmount)
+    ? formatNumber(usdAmount)
+    : absentValue;
+
+  let icpPrice: number | undefined;
+  $: icpPrice = $icpSwapUsdPricesStore?.[LEDGER_CANISTER_ID.toText()];
+
+  let icpPriceFormatted: string;
+  $: icpPriceFormatted = nonNullish(icpPrice)
+    ? formatNumber(icpPrice)
+    : absentValue;
+
+  let icpAmount: number | undefined;
+  $: icpAmount = icpPrice && usdAmount && usdAmount / icpPrice;
```

Comment:
[nitpick] is a bit easier to read
```suggestion
  $: icpAmount = icpPrice && usdAmount && (usdAmount / icpPrice);
```

Reply:
Done

Reply:
It's automatically removed by `prettier`. Sorry.